app/src/main/java/com/lpnu/mobile/MainActivity.java

```diff
@@ -12,14 +13,17 @@
 import butterknife.ButterKnife;
 
 public class MainActivity extends AppCompatActivity {
+    private boolean init;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        init = true;
         setContentView(R.layout.activity_main);
         ButterKnife.bind(this);
         AllList allItemsFragment = new AllList();
         setCurrentFragment(allItemsFragment);
+        init = false;
```

Comment:
There is an easier way of doing that. Just make change signature of your `setCurrentFragment` method to following:
`public void setCurrentFragment(Fragment fragment, boolean addToBackStack)`
And then for first fragment you just call `setCurrentFragment(allItemsFragment, false)`. Call `setCurrentFragment(allItemsFragment, true)` for any other fragment